src/Shared/Internal/Iterable/SequenceEnvelope.php

```
@@ -0,0 +1,36 @@
+     */
+    public function __construct(private Sequence $origin)
+    {
+    }
+
+    /**
+     * @return Iterator<T>
+     */
+    #[\Override]
+    public function getIterator(): Iterator
+    {
+        return $this->origin->getIterator();
```

PHPStan (2.1.15)

Method Paira\Shared\Internal\Iterable\SequenceEnvelope::getIterator() should return Iterator<mixed, T> but returns Traversable<int, T>.


(return.type)